pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -708,6 +713,35 @@ func (a *ApplicationBackupController) uploadResources(
 	return a.uploadObject(backup, resourceObjectName, jsonBytes)
 }
 
+func (a *ApplicationBackupController) uploadCRDResources(backup *stork_api.ApplicationBackup) error {
+	crdList, err := storkops.Instance().ListApplicationRegistrations()
+	if err != nil {
+		return err
+	}
+	var crds []*apiextensionsv1beta1.CustomResourceDefinition
```

Comment:
Don't need to upload all CRDs, only the ones being referenced by objects in the backup